chainerx/_fallback_workarounds.py

```diff
@@ -0,0 +1,141 @@
+# This file defines workaround implementation for
+# NumPy-compatibility functions that fall back to NumPy/CuPy functions
+# for native/cuda devices respecitvely.
+# The workaround does not support backprop, and also requires external
+# libraries mentioned above.
+# Functions defined in this file should be considered to have high priority for
+# genuine implementations.
+
+import numpy
+
+import chainerx
+
+
+try:
+    import cupy
+except Exception:
+    cupy = None
+
+
+def _to_numpy(array):
+    assert isinstance(array, chainerx.ndarray)
+    return chainerx.to_numpy(array, copy=False)
+
+
+def _from_numpy(array):
+    assert isinstance(array, numpy.ndarray)
+    return chainerx.array(array, copy=False)
+
+
+def _to_cupy(array):
+    assert cupy is not None
```

Comment:
I think this assertion may fail (although we can assume it in `_from_cupy`), i.e. ChainerX with CUDA backend is used but CuPy is unavailable?

Reply:
I must have missed the check prior to the call. This doesn't seem like an issue.